### DISCUSSION TITLE
Cannot connect to standalone MongoDB

### DISCUSSION THREAD

**Discussion by sim1986** [ant-media/Ant-Media-Server#6553] [Q&A] [answered]
I am running Ant Media in a docker container. I want to connect it to a MongoDB. I have followed the [documentation ](https://antmedia.io/databases-supported-by-ant-media-server/) on how to run the start.sh file to connect to the DB. 

When I bash into the container and use the following command:
`root@df9a4c1eb559:/# /usr/local/antmedia/start.sh -m standalone -h my_mongo_ip mymongoadminuser secret`

It tells me that DB type is redis:
`root@df9a4c1eb559:/# /usr/local/antmedia/start.sh -m standalone -h my_mongo_ip mongoadmin secret
Mode: standalone
DB type is redis
172.17.0.2 df9a4c1eb559
Running on  Linux
Starting Ant Media Server
Selected libraries: (270 items)
Root: /usr/local/antmedia`

I have also confirmed that the container can reach the mongoDB because I installed the mongo client inside the container and connected it successfully.

**Comment by burak-58** [comment]
Hi @sim1986,
Thanks for the question. Let me turn this issue into a discussion so that you can get some quick support from support team.

**Comment by yashtandon113** [comment]
Hi @sim1986 

Let us check the issue with mongodb in standalone mode on our side.

Meanwhile, you can connect to MongoDB using the cluster mode via the below command.

`sudo ./change_server_mode.sh cluster [MONGODB_SERVER_IP] [MONGODB_USERNAME] [MONGODB_PASSWORD]
`

It works fine as expected.

**Comment by muratugureminoglu** [comment, marked as answer]
Hi @sim1986 @yashtandon113 ,

I 've just sent a new PR.  

You can download  src/main/server/conf/functions.sh file and copy it to /usr/local/antmedia/conf/ and then call the ./change_server_mode.sh script again. 

https://github.com/ant-media/Ant-Media-Server/pull/6564

Thanks.
